python/python/tests/conftest.py

```diff
@@ -71,8 +85,27 @@ def call_before_all_pytests(request):
     """
     tls = request.config.getoption("--tls")
     load_module = request.config.getoption("--load-module")
-    pytest.redis_cluster = RedisCluster(tls, True, load_module=load_module)
-    pytest.standalone_cluster = RedisCluster(tls, False, 1, 1, load_module=load_module)
+    endpoints = request.config.getoption("--cluster-endpoints")
+    standalone_endpoints = request.config.getoption("--standalone-endpoints")
+
+    if endpoints or standalone_endpoints:
```

Comment:
add a comment explaining the endpoints were passed by the caller, not creating clusters internally